README.md

```diff
@@ -1,36 +1,43 @@
 # Google cloud conformance tests
-This maven module is intended to function as a central location for integration with
+This Maven project functions as a central location for integration with
 [googleapis/conformance-tests](https://github.com/googleapis/conformance-tests).
 
 In this directory `conformance-tests` which is a git-submodule, containing the reference for which
-the various conformance test resources are updated to. 
+the various conformance test resources are updated to.
+
+This project contains scripts to help with updating the resources based on the state of the
+`conformance-tests` submodule. As well as performing the heavy lifting to ensure all resources
+are in the correct locations, generate protobuf classes from proto files and represent a
+released versioned jar of the conformance tests. 
 
 ## Updating conformance-tests
 
 #### Prerequisites
 
-Part of the process of updating the conformance tests involves running maven a maven build for 
-this module. Ensure that all parent modules have been installed locally so the build can run.
-```bash
-pushd cd ../../
-mvn -Dmaven.test.skip.exec=true install
-popd
-```
+Part of the process of updating the conformance tests involves running a maven build for
+this project. Ensure that maven 3.6+ is on your `PATH`.
 
 #### Performing the update
 
-To update the conformance tests run the following commands from this module directory:
+A bash script is available to perform an update. The script can be ran for any branch, so if you
+need to update for a branch other than `master` update the following snippet with your
+branch name. 
+
+To update the conformance tests run the following commands from the repository root:
 ```bash
-pushd conformance-tests
-git pull
-popd
-./generate-conformance-tests.sh
-git add .
+git checkout master
+git pull upstream master
```

Comment:
Do I need to create a branch?

`git checkout -b branch-name`

Reply:
Oh, https://github.com/googleapis/java-conformance-tests/blob/master/bump-conformance-tests.sh#L53 creates a new branch as well. That wasn't clear in the README.md section though.

Reply:
I've tried to clarify this in the sentence right after this code block, can you see if that helps?

Reply:
Thanks, do you want to add command `git push`?

Reply:
Done